docs/en_US/getting-started/getting-started.md

```diff
@@ -1,113 +1,222 @@
-# Overview
+# Product overview
 
-This project provides an operator for managing EMQX clusters on Kubernetes.
+This project provides an Operator for managing EMQX clusters on Kubernetes.
 
-**Note**: EMQX Operator Controller requires Kubernetes v1.20.11 and up.
+## Deploy EMQX Operator
 
-## Running the Operator
+### Prepare the environment
 
-### Prepare
+Before deploying EMQX Operator, please confirm that the following components have been installed:
 
-We use a [cert-manager](https://github.com/jetstack/cert-manager) for provisioning the certificates for the webhook server. You can follow [the cert-manager documentation](https://cert-manager.io/docs/installation/) to install it.
+| Software                | Version Requirements |
+|:-----------------------:|:--------------------:|
+|  Kubernetes             |  >= 1.24             |          
+|  Helm                   |  >= 3                |
+|  cert-manager           |  >= 1.1.6            |
+
+- Why Kubernetes 1.24 is needed
+
+The `MixedProtocolLBService` feature is enabled by default in Kubernetes 1.24 and above. For its documentation, please refer to: [MixedProtocolLBService](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/). The `MixedProtocolLBService` attribute allows different protocols to be used within the same Service instance of type `LoadBalancer`. Therefore, if the user deploys the EMQX cluster on Kubernetes and uses the `LoadBalancer` type of Service, there are both TCP and UDP protocols in the Service, please pay attention to upgrading the Kubernetes version to 1.24 or above, otherwise the Service creation will fail.
+
+### Install Helm
+
+1. Install Helm
+
+Helm is a Kubernetes package management tool. Execute the following command to install Helm directly. For more Helm installation methods, please refer to: [Installing Helm](https://helm.sh/zh/docs/intro/install/).
+
+```bash
+curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+```
+
+2. Add Helm chart repository
+
+```bash
+helm repo add emqx https://repos.emqx.io/charts
+helm repo update
+```
+
+### Install cert-manager
+
+EMQX Operator uses [cert-manager](https://github.com/cert-manager/cert-manager) to provide certificates for webhook services. Deploy cert-manager with the following command:
+
+```bash
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.yaml
```

Comment:
Just make sure cert-manager is ready,I think we don't need tell user how to install cert-manager

Reply:
> Just make sure cert-manager is ready,I think we don't need tell user how to install cert-manager

ok, i will make a change